dotCMS/src/main/java/com/dotmarketing/portlets/folders/business/FolderFactoryImpl.java

```diff
@@ -219,6 +220,8 @@ protected Folder findFolderByPath(String path, final Host site) throws DotDataEx
 		if(site == null || path == null){
 			return null;
 		}
+		// replace nasty double //
+		path=path.replaceAll("//", "/");
```

Comment:
StringPool already has SLASH, feel free to add DOUBLE_SLASH

Reply:
Done